src/tests/index.test.js

```diff
@@ -0,0 +1,549 @@
+describe('Homework - 4', () => {
+    describe('Test init method', () => {
+        describe('ArrayList.init', function (){
+            it('should be defined', function() {
+                assert.isDefined(ArrayList.init, 'method has been defined')
+            });
+            it('should warning user to [] (undefined)', () => {
+                const arr = undefined;
+                const expSize = 'Please input an array';
```

Comment:
How the length should be a string ?  `const expSize = 'Please input an array';` ?